package.json

```diff
@@ -46,7 +46,7 @@
     "@babel/preset-flow": "7.18.6",
     "@babel/preset-react": "7.18.6",
     "@rollup/plugin-replace": "^2.3.1",
-    "@stripe/connect-js": "2.0.1-beta.7",
+    "@stripe/connect-js": "3.0.0-beta.1",
```

Comment:
This PR LGTM, let's just make sure to update this to the version that does not have the race conditions